Common/Securities/ErrorCurrencyConverter.cs

```diff
@@ -0,0 +1,40 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+
+namespace QuantConnect.Securities
+{
+    /// <summary>
+    /// Provides an implementation of <see cref="ICurrencyConverter"/> for use in
+    /// tests that don't depend on this behavior.
+    /// </summary>
+    public class ErrorCurrencyConverter : ICurrencyConverter
```

Comment:
Nice idea 👍

Reply:
thanks! great way to confirm we _don't_ depend on it, super useful in isolating tests that need it and those that dont